services/emailer/config.py

```diff
@@ -90,8 +90,8 @@ class Config:  # pylint: disable=too-few-public-methods
     NOTIFY_API_VERSION = os.getenv("NOTIFY_API_VERSION", "")
     NAMEX_API_URL = os.getenv("NAMEX_API_URL", "")
     NAMEX_API_VERSION = os.getenv("NAMEX_API_VERSION", "")
-    LEGAL_API_URL = os.getenv("NOTIFY_API_URL", "https://legal-api-dev.apps.silver.devops.gov.bc.ca")
-    LEGAL_API_VERSION = os.getenv("NOTIFY_API_VERSION", "/api/v2")
+    LEGAL_API_URL = os.getenv("LEGAL_API_URL", "https://legal-api-dev.apps.silver.devops.gov.bc.ca")
+    LEGAL_API_VERSION = os.getenv("LEGAL_API_VERSION", "/api/v2")
```

Comment:
This key should be `LEGAL_API_VERSION_2` here and in the example file (and maybe elsewhere).

![image](https://github.com/user-attachments/assets/357de0a6-7066-4efc-89c0-3c251244d415)

Reply:
OK, you've reverted this to v1, so the original key name is ok. Resolving.